### DISCUSSION TITLE
[Adonis 41] - How can i configure my VS Code to not add semicolon at end of each line ?

### DISCUSSION THREAD

**Discussion by wemersonrv** [adonisjs/core#1405] [Help] [answered]
Hey guys,

How can i configure my VS Code to not add semicolon at end of each line ?

I know using it or not will not change anything on my app, but i just want to make it works equal to the rest of the code created by adonis-cli.

**Comment by kennedyrs** [comment, marked as answer]
You can use Prettier 
```zsh 
yarn add -D prettier eslint-config-prettier eslint-plugin-prettier
```

then, create a prettier.config.js at the root of the project with 
```js 
module.exports = {
  //singleQuote: true,
  //trailingComma: 'all',
  //arrowParens: 'avoid',
  semi: false,
}
```

**Reply by wemersonrv** [reply]
> You can use Prettier
> 
> ```shell
> yarn add -D prettier eslint-config-prettier eslint-plugin-prettier
> ```
> 
> then, create a prettier.config.js at the root of the project with
> 
> ```js
> module.exports = {
>   //singleQuote: true,
>   //trailingComma: 'all',
>   //arrowParens: 'avoid',
>   semi: false,
> }
> ```

It works, thanks.. Amazing tip. I've used prettier before, but I didn’t remember about using it for that purpose too!
